ci/libvcx-tester.dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG ALPINE_CORE_IMAGE
```

Comment:
How much smaller is this image compared to the current libvcx image?

Reply:
1.5gb

Reply:
Wow, nice :)